Gemfile

```diff
@@ -38,7 +23,6 @@ unless dependencies.detect { |d| d.name == "manageiq-providers-azure" }
 end
 
 unless dependencies.detect { |d| d.name == "manageiq-ui-classic" }
-  #gem "manageiq-ui-classic", :path => "../manageiq-ui-classic/"
```

Comment:
LOL...I had that in a change locally that I thought I already pushed up but apparently never did... I was so confused.